ocfweb/docs/templates/partials/doc-toc.html

```diff
@@ -1,4 +1,13 @@
 {% if html %}
-    <h3>Table of Contents</h3>
-    {{html|safe}}
+  <div class="ocf-doc-toc">
+    {% if collapsible %}
+        <a data-toggle="collapse" data-target="#toc" class="btn btn-lg">Table of Contents</a>
+        <div id="toc" class="collapse">
+          {{html|safe}}
+        </div>
```

Comment:
This works great. It'd be cool if it showed a + / - icon to indicate what it does, though.

What about something like...

```patch
diff --git a/ocfweb/docs/templates/partials/doc-toc.html b/ocfweb/docs/templates/partials/doc-toc.html
index c851ffb..9a2aad1 100644
--- a/ocfweb/docs/templates/partials/doc-toc.html
+++ b/ocfweb/docs/templates/partials/doc-toc.html
@@ -1,7 +1,10 @@
 {% if html %}
   <div class="ocf-doc-toc">
     {% if collapsible %}
-        <a data-toggle="collapse" data-target="#toc" class="btn btn-lg">Table of Contents</a>
+        <a data-toggle="collapse" data-target="#toc" class="doc-collapse-toggle collapsed">
+            <span class="glyphicon" aria-hidden="true"></span>
+            Table of Contents
+        </a>
         <div id="toc" class="collapse">
           {{html|safe}}
         </div>
diff --git a/ocfweb/static/scss/pages/docs.scss b/ocfweb/static/scss/pages/docs.scss
index 13ab148..2732c15 100644
--- a/ocfweb/static/scss/pages/docs.scss
+++ b/ocfweb/static/scss/pages/docs.scss
@@ -54,6 +54,25 @@
     }
 }
 
+.doc-collapse-toggle {
+    @extend .btn;
+    @extend .btn-lg;
+
+    .glyphicon {
+        @extend .glyphicon-minus;
+
+        font-size: 10px;
+        position: relative;
+        top: -2px;
+        margin-right: 2px;
+    }
+
+
+    &.collapsed .glyphicon {
+        @extend .glyphicon-plus;
+    }
+}
+
 .ocf-sidebar .ocf-doc-toc {
     display: inline;
 }
```

Not sure if there is a better way to do this, but it seems to work.